docs/whats-new/release-notes/v2_14_0.md

```diff
@@ -47,18 +48,36 @@ Zowe Version 2.14.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
+- Corrected extra character being displayed at the end of lines when issuing command `zowe files compare` on Windows. [#1992](https://github.com/zowe/zowe-cli/issues/1992)
+- Corrected the online help description for `zowe files compare uss`. [#1754](https://github.com/zowe/zowe-cli/issues/1754)
+- Fixed typo in command help for `zowe zos-workflows create` commands.
```

Comment:
@adam-wolfe Can we add a PR/issue link to this?

Reply:
https://github.com/zowe/zowe-cli/pull/2003